specs/AppLifecycle/EnvironmentVariables/AppLifecycle - Environment Variables (EV).md

```diff
@@ -310,27 +302,17 @@ packaged, writes to HKCU are virtualized, unless the app has the unvirtualizedRe
 restricted capability and has suppressed registry write virtualization for HKCU.
 
 An unpackaged or packaged Win32 (Desktop Bridge) app can write to HKLM if the
-app is running elevated. A UWP app cannot run elevated. In the normal case,
+app is running elevated. In the normal case,
 where virtualization is not disabled, all writes to HKCU actually go to a
```

Comment:
This is confusing since this is about both packaged and unpackaged. Also using a double-negative (not disabled). I think the deleted table was clearer. Do we even need to list the implementation detail of where the private hive is located?

Reply:
I removed both paragraphs.  I kept the "Moving forward, the new APIs..." table.  The  text I deleted wasn't directly related to EM, and more "Here is windows' implementation."